app.rb

```diff
@@ -202,29 +202,34 @@ def write_csv_for_mturk(path, all_assignments, approved_assignments, rejected_as
 assignments_to_approve = Set.new
 assignments_to_reject = {}
 
+columns = {}
+
 is_header = true
 CSV.foreach('batch.csv') do |row|
-  # Skip header row
```

Comment:
We aren't skipping the header row anymore. Now we save the index of each column of the header. https://github.com/hackclub/process-mturk-results/pull/1/files#diff-cc95738088603531796e0d0f246a5d77R211